content/en/data_jobs/databricks.md

```diff
@@ -71,6 +75,10 @@ Datadog can install and manage a global init script in the Databricks workspace.
 
 {{% tab "Manually install a global init script" %}}
 
+<div class="alert alert-warning">
+This setup does not support Databricks Clusters on Shared Policy. If you are using Shared Clusters, you must follow the <a href="?tab=manuallyinstallonaspecificcluster#install-the-datadog-agent">Manually install on a specific cluster</a> instructions instead.
```

Comment:
```suggestion
This setup does not work on Databricks Clusters in `Standard (formerly Shared)` access mode because global init scripts cannot be installed on those clusters. If you are using clusters with the `Standard (formerly Shared)` access mode, you must follow the <a href="?tab=manuallyinstallonaspecificcluster#install-the-datadog-agent">Manually install on a specific cluster</a> instructions for installation on those specific clusters.
```